letsencrypt/cli.py

```diff
@@ -1139,6 +1142,27 @@ def main(cli_args=sys.argv[1:]):
 
     return args.func(args, config, plugins)
 
+def check_config_sanity(args):
```

Comment:
I guess that's probably right.  Even if it's called from cli.py rather than the NamesspaceConfigurator constructor, it makes logical sense for it to be a proprety of configurations.